frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mention/mention_date_block.dart

```diff
@@ -264,10 +264,10 @@ class _MentionDateBlockState extends State<MentionDateBlock> {
 
     try {
       if (timeFormat == TimeFormatPB.TwelveHour) {
-        return twelveHourFormat.parse(timeStr);
+        return twelveHourFormat.parseStrict(timeStr);
```

Comment:
Do we still need to use parseStrict?

Reply:
Without ParseStrict the Time string can have a value greater than 24. Which will be confusing.

For example :
36:50 gets parsed into 12:50
40:90 into 17:30

If it is preferred this way, we can change it back to `parse` again